spinalcordtoolbox/flattening.py

```diff
@@ -35,7 +35,8 @@ def flatten_sagittal(im_anat, im_centerline, verbose):
     nx, ny, nz, nt, px, py, pz, pt = im_anat.dim
 
     # smooth centerline and return fitted coordinates in voxel space
-    _, arr_ctl, _, _ = get_centerline(im_centerline, param=ParamCenterline(), verbose=verbose)
+    _, centerline, _ = get_centerline(im_centerline, param=ParamCenterline(), verbose=verbose)
+    arr_ctl = centerline.arr_ctl
```

Comment:
I don't think it is useful to keep arr_ctl

Reply:
I agree!

My only reason for including the redundant variables in commit [784704c](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4010/commits/784704c4a15f24a3f9cd54c2d49cca976cf88f69) was to make it very clear to reviewers that the changes preserved the existing functionality.

But, as you point out, there is definitely some cleanup that can be done.

Reply:
This concern has been addressed, because the highlighted changes have been reverted in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4010#discussion_r1108902353.